packages/database/src/__tests__/changeHandlers/SurveyResponseOutdater.test.js

```diff
@@ -76,18 +76,19 @@ describe('SurveyResponseOutdater', () => {
 
     for (const [surveyResponseId, expected] of Object.entries(expectedByResponseId)) {
       const surveyResponse = surveyResponses.find(r => r.id === surveyResponseId);
-      const survey = await models.survey.findById(surveyResponse.survey_id);
-      const entity = await models.entity.findById(surveyResponse.entity_id);
-      const responseDescriptionFields = {
-        survey_code: survey.code,
-        entity_id: entity.code,
-        data_time: surveyResponse.data_time,
-      };
-      const message = `Failed assertion for survey response ${JSON.stringify(
-        responseDescriptionFields,
-      )}`;
-
-      expect(surveyResponse).toHaveProperty('outdated', expected, message);
+      // TODO use custom assertion message when jest-expect-message is re-enabled
+      // const survey = await models.survey.findById(surveyResponse.survey_id);
+      // const entity = await models.entity.findById(surveyResponse.entity_id);
+      // const responseDescriptionFields = {
+      //   survey_code: survey.code,
+      //   entity_id: entity.code,
+      //   data_time: surveyResponse.data_time,
+      // };
+      // const message = `Failed assertion for survey response ${JSON.stringify(
+      //   responseDescriptionFields,
+      // )}`;
+
+      expect(surveyResponse).toHaveProperty('outdated', expected);
```

Comment:
Using a 3rd parameter (for a custom message) is not only unsupported by `jest`, but it causes `toHaveProperty` to always pass! :fearful:

Interestingly this is not the case with `toBe`, which fails as expected. I double-checked that the only instance where custom messages are still used in this package is in [AnalyticsRefresher.test.js](https://github.com/beyondessential/tupaia/pull/4188/files#diff-ae6df06af6401ae30c75e21dac78e71ba688d5`7de5edf2001266e22f8e3b4f8e), for `.toBe` assertions.